src/components/Bookmark/Bookmark.spec.js

```diff
@@ -0,0 +1,12 @@
+import {render, screen} from '@testing-library/react';
+import Bookmark from './Bookmark';
+
+describe('Bookmark', () => {
+  it('renders the Bookmark component', () => {
+    render(<Bookmark />);
+
+    const button = screen.getByRole('button');
+
+    expect(button).toBeInTheDocument();
+  });
```

Comment:
If you are really tough, you could add another test case and check whether the `onBookmark` function is called ;) But this is for another PR, I guess.